README.md

```diff
@@ -1 +1,5 @@
-# .github
\ No newline at end of file
+![test](https://github.com/renovatebot/.github/workflows/test/badge.svg)
+
+# .github
+
+This repository contains the basic Renovate configuration that is used in most/all other Renovate repositories.
```

Comment:
```suggestion
This repository contains the basic Renovate configuration that is used in most/all other renovatebot repositories.
```

Should probably be `renovatebot` instead of `Renovate`?